Do not warn about madvise not being implemented. It does not have any real
semantic anyway, and that brings some -Werror FTBFS.

---
 misc/madvise.c        |    4 ++--
 posix/posix_madvise.c |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

--- a/misc/madvise.c
+++ b/misc/madvise.c
@@ -30,5 +30,5 @@
   return -1;
 }
 libc_hidden_def (madvise)
-stub_warning (madvise)
-#include <stub-tag.h>
+/* stub_warning (madvise)
+#include <stub-tag.h> */
--- a/posix/posix_madvise.c
+++ b/posix/posix_madvise.c
@@ -28,5 +28,5 @@
 {
   return ENOSYS;
 }
-stub_warning (posix_madvise)
-#include <stub-tag.h>
+/* stub_warning (posix_madvise)
+#include <stub-tag.h> */
